test/metabase/driver/postgres_test.clj

```diff
@@ -1356,3 +1357,42 @@
     ;; Special characters
     (is (= "SET ROLE \"Role.123\";"   (driver.sql/set-role-statement :postgres "Role.123")))
     (is (= "SET ROLE \"$role\";"      (driver.sql/set-role-statement :postgres "$role")))))
+
+(deftest get-tables-parity-with-jdbc-test
```

Comment:
Nice test. Is this going to be removed afterwards, or do you think it's worth keeping around? It will make it harder to extend `get-tables` in the future to have more features, because this test will fail.

Reply:
I think it's nice to keep it. for future use cases, we are probably going to add more fields to table metadata; in those cases, we can dissoc the new keys for this test